src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -90,7 +90,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('no_wait', help="Do not wait for the run to complete and return immediately after queuing the run.", action='store_true')
         c.argument('no_format', help="Indicates whether the logs should be displayed in raw format", action='store_true')
         c.argument('os_type', options_list=['--os'], help='The operating system type required for the build.', arg_type=get_enum_type(OsType), deprecate_info=c.deprecate(redirect='platform', hide=True))
-        c.argument('platform', help="The platform of the resulting image in the format of 'os/arch/variant'. E.g. linux/arm/v7. The 'arch' and 'variant' parts are optional.")
+        c.argument('platform', help="The platform where build/task is run, Eg, 'windows' and 'linux'. When it's used in build commands, it also can be specified in 'os/arch/variant' format for the resulting image. Eg, linux/arm/v7. The 'arch' and 'variant' parts are optional.")
```

Comment:
I think we can be more specific like "When it's used in `ac acr build command`, it can...."